gspread/worksheet.py

```diff
@@ -244,14 +291,14 @@ def cell(self, row, col, value_render_option=ValueRenderOption.formatted):
                 rowcol_to_a1(row, col), value_render_option=value_render_option
             )
 
-            value = data.first()
+            value = data.first() or ""
```

Comment:
why this change?

Reply:
I see. It makes sense to me to cast the value to a string with `str(value)`. Otherwise, the `or ""` exists purely for type-checking purposes, and doesn't actually do anything.

However, the return type of `first` is "str" or "None", so with that it makes sense. But... *is* the return type of `first` *always* "str" or "None"?

Reply:
I need to check that, let's cast it to string first. The `value` of a `Cell` object is set to `str` for now I'll update it if I can find some code path that can set it to any value (like `int` etc..)